whitepaper/whitepaper.md

```diff
@@ -124,12 +133,248 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
 
 We propose a protocol and it's implementation to connect as many of them as possible in an attempt to solve the fragmentation problem by creating a heterogeneous system architecture <sup>[1](#7-references)</sup>.
 
+
+
+There are two inherent problems<sup>[a](#12-footnotes)</sup> that have to be solved when connecting different blockchains: 
+* How to retrieve the
+networkwide<sup>[b](#12-footnotes)</sup> ledger view aka the state of a connected blockchain from the outside?
+* How can other entities verify a given claim about the state of a connected blockchain
+from the outside? 
+
+`Cactus` operates several validator nodes above the
```

Comment:
I thought this is also need to clarify your expectations. My assumed the relationship between Cactus SDK and Validator is following: 
- The Validator will provide the proof on a transaction is observed at the Ledger.
- The SDK (at business logic plugin) can complete set of transactions or issue another transaction to Ledger
Does it make sense to you ?